developer-docs-site/docs/standards/object.md

```diff
@@ -0,0 +1,239 @@
+LiquidityPool resource is part of the ObjectGroup resource group. This means that the LiquidityPool resource is stored in the same storage slot as the ObjectCore resource. This is more storage and gas efficient for reading and writing data.
+
+LiquidityPool resource can be added during construction of the object:
+```rust
+pub fun create_liquidity_pool(
+    token_a: Object<FungibleAsset>,
+    token_b: Object<FungibleAsset>,
+    reserves_a: u128,
+    reserves_b: u128
+): Object<LiquidityPool> {
+    let exchange_signer = &get_exchange_signer();
+    let liquidity_pool_constructor_ref = &object::create_object_from_account(exchange_signer);
+    let liquidity_pool_signer = &object::generate_signer(liquidity_pool_constructor_ref);
+    move_to(liquidity_pool_signer, LiquidityPool {
+        token_a: token_a,
+        token_b: token_b,
+        reserves_a: reserves_a,
+        reserves_b: reserves_b
+    });
+}
+```
+
+[liquidity_pool.png](../../static/img/docs/liquidity_pool.png)
+
+More resources can also be added post-creation if the exchange module stores the ExtendRef. This is covered in more detail in the Capabilities section.
+
+## Object Lifecycle
+### Creation
+Objects can be created via several different functions provided in the object module:
+```rust
+/// Create a new named object and return the ConstructorRef. Named objects can be queried globally
+/// by knowing the user generated seed used to create them. Named objects cannot be deleted.
+public fun create_named_object(creator: &signer, seed: vector<u8>): ConstructorRef;
+
+/// Create a new object from a GUID generated by an account.
+public fun create_object_from_account(creator: &signer): ConstructorRef;
+
+/// Create a new object from a GUID generated by an object.
+public fun create_object_from_object(creator: &signer): ConstructorRef;
+```
+
+These functions generate object addresses in different schemas:
+1. create_named_object generates an address from the caller-provided seed and creator address. This is a deterministic address that can be queried globally. The formula used is sha3(creator address + seed + 0xFD).
+2. create_object_from_account generates an address from the caller's address and a GUID generated by the caller's account. The formula used is sha3(creator address + account guid + 0xFD).
+3. create_object_from_object generates an address from the caller's address and a GUID generated by the caller's object. The formula used is sha3(creator address + object guid + 0xFD).
+The domain separation ensures there's no conflict among objects created via these different functions.
+
+Note that since named objects have deterministic addresses, they cannot be deleted. This is to prevent a malicious user from creating an object with the same seed as a named object and deleting it.
+
+### Object capabilities (refs)
```

Comment:
let's avoid mentioning caps... with refs... I think we'd better stick to refs only.